ext/QuantumSavoryMakie/QuantumSavoryMakie.jl

```diff
@@ -313,8 +313,9 @@ function registernetplot_axis(ax::Makie.AbstractAxis, registersobservable; infoc
     ax.parent, ax, p, p[1]
 end
 
-function registernetplot_axis(subfig::Makie.GridPosition, registersobservable; infocli=true, datainspector=true, kwargs...)
-    registernetplot_axis(Makie.Axis(subfig), registersobservable; infocli, datainspector, kwargs...)
+# subfig::Union{GridPosition, GridSubposition} but maybe other as well, so leave it unspecified
+function registernetplot_axis(subfig, registersobservable; infocli=true, datainspector=true, kwargs...)
+    registernetplot_axis(Makie.Axis(subfig[1,1]), registersobservable; infocli, datainspector, kwargs...)
```

Comment:
@hanakl , FYI about this change. I made it as  otherwise it was failing to render on subfigures.